src/turbomind/models/llama/LlamaBatch.cc

```diff
@@ -743,6 +767,12 @@ void LlamaBatch::AllocateBuffer(ssize_t batch_size, ssize_t session_len, int cac
 
         s.curand_state = {{batch_size, sizeof(curandState_t)}, kDEVICE};
         Clear(s.curand_state.buffer());
+
+        if (model_->attn_param_.rope.type == RopeType::kMultimodal) {
+            s.mrope.position_ids   = {{batch_size, session_len_ * 3}, kDEVICE};
```

Comment:
Can we use shape `[batch_size, session_len, 3]` to avoid the magic number 3 from recurring?